packages/react-app-revamp/lib/contests/index.ts

```diff
@@ -28,9 +28,9 @@ async function getContractConfig(address: string, chainName: string, chainId: nu
   return contractConfig;
 }
 
-const fetchTokenBalances = async (contest: any, contestRewardModuleAddress: string) => {
+export const fetchTokenBalances = async (chainName: string, contestRewardModuleAddress: string) => {
   try {
-    const networkName = contest.network_name.toLowerCase() === "arbitrumone" ? "arbitrum" : contest.network_name;
```

Comment:
oo, small thing here actually - in #402 we update our arbitrum chain config to be arbitrumone, so we don't need this check anymore!

Reply:
do you only mean for the `toLowerCase()` part? since alchemy still perceives it as `arbitrum` ?

Reply:
Oh no, in #402 we changed it so that alchemy/the provider is named `arbitrumone`, so we don't need to do this check.